pyproject.toml

```
@@ -1,6 +1,268 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = [
+    "hatchling>=1.2.0"
+]
+
+[project]
+name = "elyra"
+description="Elyra provides AI Centric extensions to JupyterLab"
+maintainers = [
+    {name = "Elyra Maintainers"},
+]
+readme = "README.md"
+requires-python = ">=3.7"
+dependencies = [
+    "autopep8>=1.5.0",
+    "click>=8",  # elyra-ai/elyra#2579
+    "colorama",
+    "deprecation",
+    "entrypoints>=0.3",
+    "jinja2>=3",
+    "jsonschema>=3.2.0,<4.0",  # Cap from kfp
+    "jupyter_core>=4.6.0",
+    "jupyter_client>=6.1.7",
+    "jupyter-packaging>=0.10",
+    "jupyter_server>=1.7.0",
+    "jupyterlab>=3.4.6",  # comment out to use local jupyterlab
+    "jupyterlab-lsp>=3.8.0",  # comment out to use local jupyterlab
+    "jupyterlab-git~=0.32",  # Avoid breaking 1.x changes
+    "jupyter-resource-usage>=0.5.1",
+    "MarkupSafe>=2.1",
+    "minio>=7.0.0",
+    "nbclient>=0.5.1",
+    "nbconvert>=6.4.5",
+    "nbdime~=3.1",  # Cap from jupyterlab-git
+    "nbformat>=5.1.2",
+    "networkx>=2.5.1",
+    "papermill>=2.3.4",
+    "python-lsp-server[all]>=1.1.0",
+    "pyyaml>=5.3.1,<6.0",  # Cap from kfp
+    "requests>=2.25.1",
+    "rfc3986-validator>=0.1.1",
+    "tornado>=6.1.0",
+    "traitlets>=4.3.2",
+    "typing-extensions>=3.10,<5",  # Cap from kfp
+    "urllib3>=1.26.5",
+    "watchdog>=2.1.3",
+    "websocket-client",
+    "yaspin",
+    "kfp>=1.7.0,<2.0,!=1.7.2",  # We cap the SDK to <2.0 due to possible breaking changes
+    "pygithub",
+    "black<=21.12b0",  # Cap due to psf/black#2846
+]
+classifiers = [
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Looks like this change from #3033 wasn't applied.
```suggestion
    "Programming Language :: Python :: 3.10",
    "Programming Language :: Python :: 3.11",
```